src/main/java/com/conveyal/r5/analyst/GridComputer.java

```diff
@@ -188,20 +189,32 @@ public void run() throws IOException {
 
```

Comment:
LinkedPointSet linkedDestinationsDirect = destinations.link(network.streetLayer, accessMode);
I'm not sure if this uses correct accessMode since currently if CAR_PARK is used accessMode is WALK.